usr/share/rear/conf/default.conf

```diff
@@ -373,8 +373,11 @@ LANG_RECOVER=C
 FDRUPSTREAM_INSTALL_PATH="/opt/fdrupstream"
 COPY_AS_IS_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH" )
 COPY_AS_IS_EXCLUDE_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/usserver" )
-#CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/upstream.cfg" "$FDRUPSTREAM_INSTALL_PATH/uscmd1" "$FDRUPSTREAM_INSTALL_PATH/usdaemon2" )
+CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/uscmd*" "$FDRUPSTREAM_INSTALL_PATH/usdaemon*" "$FDRUPSTREAM_INSTALL_PATH/us" "$FDRUPSTREAM_INSTALL_PATH/us1" "$FDRUPSTREAM_INSTALL_PATH/us.ser" "$FDRUPSTREAM_INSTALL_PATH/*.so" "$FDRUPSTREAM_INSTALL_PATH/usudb*" "$FDRUPSTREAM_INSTALL_PATH/*.dat" "$FDRUPSTREAM_INSTALL_PATH/*.cfg" )
```

Comment:
Are you aware of the fact that the user won't be able to change the `FDRUPSTREAM_INSTAL_PATH`? If he does all the other `*FDRUPSTREAM*` variables are already set based on the default path. If you really want to enable users to change that then you will have to move this code to the `prep` stage for example. If you just did not want to repeat the path all the time then it will work. In that case I would recommend marking the variable as readonly so that a user will get an error if he tries setting the variable.